src/pages/DelegatoryValidator/DetailCard.tsx

```diff
@@ -76,18 +88,21 @@ export default function ValidatorDetailCard({
           }
         />
         <ContentRow
+          container={false}
           sx={sx}
           title="Rewards Performance"
           value={rewardGrowth ? `${rewardGrowth.toFixed(2)} %` : null}
           tooltip={<RewardsPerformanceTooltip />}
         />
         <ContentRow
+          container={false}
           sx={sx}
           title="Last Epoch Performance"
           value={validator ? validator.last_epoch_performance : null}
           tooltip={<LastEpochPerformanceTooltip />}
         />
         <ContentRow
+          container={false}
```

Comment:
for the details card below, we don't want container effect but rather have `space-between` effect: 
<img width="564" alt="Screenshot 2023-01-24 at 2 52 42 PM" src="https://user-images.githubusercontent.com/121921928/214438968-c717bff4-ac50-4984-80ae-a4e132015ab4.png">